lib/puppet/provider/cs_order/pcs.rb

```diff
@@ -19,37 +19,25 @@ def self.instances
     instances = []
 
     cmd = [ command(:pcs), 'cluster', 'cib' ]
-    raw, status = run_pcs_command(cmd)
+    raw, _ = run_pcs_command(cmd)
     doc = REXML::Document.new(raw)
 
     doc.root.elements['configuration'].elements['constraints'].each_element('rsc_order') do |e|
       items = e.attributes
 
-      if items['first-action']
-        first = "#{items['first']}:#{items['first-action']}"
-      else
-        first = items['first']
-      end
+      first = items['first']
+      second = items['then']
```

Comment:
Was planning to add this as a separate pull request. I think actions should be their own parameters, rather than set both first and firstaction as one parameter. 
```puppet
first => 'foo:promote',
```
Would become
```puppet
first => 'foo',
firstaction => 'promote',
```
Should this be part of this pull request? Alternatively leave it as is and just append :start as a default to first => 'foo'